src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/DesignerUtils.cs

```diff
@@ -178,10 +184,10 @@ public static void SyncBrushes()
             s_grabHandleFillBrush = Gdi32.CreateSolidBrush(ColorTranslator.ToWin32(SystemColors.ControlText));
 
             Gdi32.DeleteObject(s_grabHandlePenPrimary);
-            s_grabHandlePenPrimary = Gdi32.CreatePen(Gdi32.PS.SOLID, 1, ColorTranslator.ToWin32(SystemColors.ControlText));
+            s_grabHandlePenPrimary = PInvoke.CreatePen(Gdi.PEN_STYLE.PS_SOLID, 1, (uint)ColorTranslator.ToWin32(SystemColors.ControlText));
```

Comment:
we adapted adding parameter names for constants in the parameter list. It would be nice to add those for the lines you are touching here and other parts of this PR. Or do you see any issues with that in PInvokes? ex:
```suggestion
            s_grabHandlePenPrimary = PInvoke.CreatePen(Gdi.PEN_STYLE.PS_SOLID, cWidth: 1, (uint)ColorTranslator.ToWin32(SystemColors.ControlText));
```

Reply:
We only need to add them where it is not super clear. That typically means where you explicitly pass `true`, `false`, or `null`. Here it is pretty obvious that you're referring to a width.

Reply:
Are we saying numeric constants does not need parameter names here? I may not agree on the part that "it is super clear" here.